ouroboros-consensus-diffusion/test/mock-test/Test/ThreadNet/BFT.hs

```diff
@@ -28,6 +28,7 @@ import           Test.ThreadNet.Util.SimpleBlock
 import           Test.Util.HardFork.Future (singleEraFuture)
 import           Test.Util.Orphans.Arbitrary ()
 import           Test.Util.Serialisation.Roundtrip
+import           Test.Util.Serialisation.SomeResult ()
```

Comment:
I'm slightly surprised this is necessary, since the `SomeResult` module contains no orphan instances. What happens if you remove it?

Reply:
Good catch!